app/controllers/identify_controller.rb

```diff
@@ -100,6 +101,10 @@ def allowed_referrer?(uri)
   end
 
   def log_certificate(cert)
+    stdout, stderr, status = Open3.capture3("openssl verify -purpose sslclient -inhibit_any -explicit_policy -CAfile #{IdentityConfig.store.login_certificate_bundle_file} -policy_check -policy 2.16.8.40.1.101.3.2.1.3.7 -policy 2.16.840.1.101.3.2.1.3.13 -policy 2.16.840.1.101.3.2.1.3.15 -policy 2.16.840.1.101.3.2.1.3.16 -policy 2.16.840.1.101.3.2.1.3.18 -policy 2.16.840.1.101.3.2.1.3.41", stdin_data: cert.to_pem)
```

Comment:
can we splat strings instead of interpolating, for safety?
```suggestion
    stdout, stderr, status = Open3.capture3("openssl", "verify", "-purpose", "sslclient", "-inhibit_any", "-explicit_policy", "-CAfile", IdentityConfig.store.login_certificate_bundle_file, ...
```

Reply:
Also do we want to do like a `Rails.root.join` or a `File.expand_path` on the path to the config file?

Reply:
Yep, went with `Rails.root.join`

Reply:
Also converted to comma-separated parameters/options